airflow_aws_executors/batch_executor.py

```diff
@@ -118,23 +139,43 @@ def _describe_tasks(self, job_ids) -> List[BatchJob]:
             all_jobs.extend(describe_tasks_response['jobs'])
         return all_jobs
 
-    def execute_async(self, key: TaskInstanceKeyType, command: CommandType, queue=None, executor_config=None):
+    def execute_async(
+        self,
+        key: TaskInstanceKeyType,
+        command: CommandType,
+        queue: Optional[str] = None,
+        executor_config=None
+    ):
         """
         Save the task to be executed in the next sync using Boto3's RunTask API
         """
+
+        # We know we will always have a queue here but we keep the same signature
+        # of the BaseExecutor so make the type system happy by asserting its not None
+        assert queue is not None
```

Comment:
Please delete the default value from the keyword arg instead of having this line. They would both result in an exception, but at least the reason for failure would be more clear.